### DISCUSSION TITLE
Dynamic Database Connection String

### DISCUSSION THREAD

**Discussion by emmanuel-londono** [prisma/prisma#21534] [Q&A] [answered]
### Question

Is it possible to set my postgres database ```connection string``` dynamically, based off user client input? The purpose of this is to switch between multiple databases according to unique user access. User 1, 2, 3 & 4 have access to db1 while users 5,6,7,8 have access to db2., for example.

What if all databases used the same prisma.schema file? Would this make the dynamic switching more feasible?

**Comment by janpio** [comment, marked as answer]
You can either change the env variable depending on the input that is defined in the schema file, or use https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#datasourceurl to override it on runtime.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
